Add airstack CLI tool for developer convenience

andrewjong opened the pull request:
This PR adds a new `airstack` CLI tool to simplify common development tasks.

### Features:

- **Installation**: `./airstack.sh install` installs Docker Engine, Docker Compose, etc.
- **Setup**: `./airstack.sh setup` configures private keys and adds the `airstack` tool to shell profile
- **Container Management**: `airstack up`, `airstack connect`, `airstack down` etc. for managing Docker containers
- **Smart Container Matching**: Partial name matching for containers with interactive selection when multiple matches are found
- **Extensible Architecture**: Modular design allows easy addition of new commands

### Implementation Details:

- Written in Bash for maximum compatibility
- Includes comprehensive help system
- Supports multiple operating systems
- Provides detailed documentation in `.airstack/README.md`

andrewjong commented:
FYI @jfkeller @adityarauniyar , this integrates in the previous ./configure.sh and WINTAK installation scripts as well. Now it's all together within the airstack cli tool.

jfkeller commented:
The readme mentions `launch` but I think it has been replaced with `up`.

I think there might be issues if there are multiple installations of airstack. I often have multiple for my own development, checking the most recent branch, and checking pull requests. When I cloned two airstack repos, the install script on the second didn't modify the bashrc so I think it the `airstack` command is looking in the first repo. Maybe it isn't an issue if people aren't expected to have multiple instances.

I didn't try the wintak related commands.

andrewjong commented:
Thanks John!

Fixed the README.

For multiple installations, I've updated the `airstack` function to search up the directory tree for `airstack.sh`. This way you can have multiple repositories installed and it will call the script that's part of the repo you're under.

andrewjong commented:
John approved over Slack, merging.
